testsuite/setup.py

```diff
@@ -123,6 +123,8 @@
                   'data/dlpoly/CONFIG*',
                   'data/dlpoly/HISTORY*',
                   'data/*.xml',
+                  'coordinates/data/*xyz',
```

Comment:
I moved them all to a subfolder in the general test data folder. It really makes a lot of sense there since everything is together. The subfolder also gives a nice logical grouping of the data.